cpu/sam0_common/periph/gpio.c

```diff
@@ -56,6 +56,14 @@
 #define _EIC EIC
 #endif
 
+/**
+ * @brief   Clock source for the External Interrupts Controller
```

Comment:
'Interrupts' -> 'Interrupt'

Reply:
Good catch

Reply:
I changed this within the squashing process.
In the end, this is the only last-minute change that went into the code. All other changes have been presented by fixups.